fastlane/swift/LaneFileProtocol.swift

```diff
@@ -137,7 +141,9 @@ open class LaneFile: NSObject, LaneFileProtocol {
         _ = fastfileInstance.perform(NSSelectorFromString(laneMethod), with: parameters)
 
         // Call only on success.
-        fastfileInstance.afterAll(with: lane)
+        if !(Self.onErrorCalled[lane] ?? false) {
```

Comment:
If we decided to use a `Set` for example this will become:

```suggestion
        if !(Self.onErrorCalled.contains(lane)) {
```